.tekton/downstream-ImageContentSourcePolicy.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
```

Comment:
As I mentioned in the [JIRA comment](https://issues.redhat.com/browse/ROX-26601?focusedId=25967792&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25967792), ICSP should better go next to other installation instructions which are in <https://spaces.redhat.com/display/StackRox/How+to+deploy+ACS+with+Konflux+images>.

This file placed here does not have any traceable connection to anything and is in fact misguiding: PaC reads yaml files in `.tekton/` directory by convention, and somebody new may think that this file is also recognized and used by PaC in some way. Whereas it is not. The problem of convention-based approach is that it's hard to find out what is the responsible system, what it does, and whether it reads this file or it's insignificant.

Reply:
Ok, I will add instructions to Confluence. ICSP is now removed from this PR.

Reply:
New instructions for deploying standalone bundles have been added to Confluence.